### DISCUSSION TITLE
The Future of EdgeX Snaps

### DISCUSSION THREAD

**Discussion by jpwhitemn** [edgexfoundry/community#183] [Polls]
EdgeX’ers, the TSC is considering the status and future of the EdgeX Snaps.   While the number of downloads encourages the project's continued support for EdgeX Snaps, the number is not high enough to warrant the project's direct role of creating Snap artifacts and putting them into the Snap Store without additional resourcing help.  We are exploring a few options.  We would appreciate your assistance in helping to determine the future of the EdgeX Snaps by answering this quick poll question.

**Comment by JamesKButcher** [comment]
Hi Folks

Just a bump to see if we can gain any comment from community users here? If not, the TSC-recommended approach would be to discontinue the creation of new EdgeX Snaps going forward and likely rely on documentation/examples for users to build their own Snaps if needed.

Thanks
James Butcher
EdgeX TSC Chairman

**Reply by farshidtz** [reply]
@outbackdingo The project only provides arm64 and amd64 releases: https://docs.edgexfoundry.org/3.0/general/PlatformRequirements/
This means you can run the amd64 releases on Raspberry Pi [Zero 2, 3, 4](https://www.raspberrypi.com/news/raspberry-pi-os-64-bit/), as well as 5. Needless to say, you need to run a 64bit OS to make that work.

Regarding the example/documentation issue, I don't see how that is relevant to snaps. If you need help, open a separate discussion post. If you think the documentation has an issue, please report it at https://github.com/edgexfoundry/edgex-docs
